docs/release-notes/release-notes-0-17-17.md

```diff
@@ -45,6 +45,26 @@ To maintain compatibility, the old window names will remain valid.
 
 Pages in [Window Types](../cep/windows/window-types/) were updated as a result of this change.
 
+### Query Worker Permissions
+
+In previous versions of Macrometa, only the user that created the query worker could directly view and edit that query worker.
+
+The 17.17 release allows all users with the appropriate access level to see and interact with query workers.
+
+The user and API permissions in the web console have been updated to allow you to assign query worker access levels.
```

Comment:
Should we say - `The user and API key permissions in the web console have been updated to allow you to assign query worker access levels.`

Reply:
Good edit, updated.